src/data_info/services/sharing.clj

```diff
@@ -148,7 +148,7 @@
 (defn anon-file-url
   [p]
   (let [aurl (url/url (cfg/anon-files-base))]
-    (str (-> aurl (assoc :path (ft/path-join (:path aurl) (string/replace p #"^\/" "")))))))
+    (str (-> aurl (assoc :path (apply ft/path-join (:path aurl) (map url/url-encode (string/split (string/replace p #"^\/" "") #"\/"))))))))
```

Comment:
This is almost too trivial to mention, but it's not necessary to escape the forward slashes.